projects/kubernetes-sigs/image-builder/Makefile

```diff
@@ -124,16 +153,27 @@ release-ami-ubuntu-2004-%: setup-ami-share deps-ami setup-packer-configs
 
 .PHONY: release-ova-ubuntu-2004
 release-ova-ubuntu-2004: MAKEFLAGS=
+release-ova-ubuntu-2004: export IMAGE_FORMAT=ova
 release-ova-ubuntu-2004: PACKER_TYPE_VAR_FILES=$(PACKER_OVA_VAR_FILES)
 release-ova-ubuntu-2004: deps-ova setup-vsphere setup-packer-configs
 	PACKER_FLAGS="-force" PACKER_LOG=1 PACKER_LOG_PATH=$(ARTIFACTS_PATH)/packer.log PACKER_VAR_FILES="$(PACKER_VAR_FILES)" \
 		OVF_CUSTOM_PROPERTIES="$(FULL_OUTPUT_DIR)/config/ovf_custom_properties.json" \
 		$(MAKE) -C $(IMAGE_BUILDER_DIR) build-node-ova-vsphere-ubuntu-2004
 
 .PHONY: release-ova-bottlerocket
+release-ova-bottlerocket: export IMAGE_FORMAT=ova
 release-ova-bottlerocket: $(TUFTOOL_TARGET) $(BOTTLEROCKET_SETUP_TARGET)
 	build/get_bottlerocket_artifacts.sh $(RELEASE_BRANCH) bottlerocket $(BOTTLEROCKET_DOWNLOAD_PATH) $(CARGO_HOME) $(PROJECT_PATH)/$(RELEASE_BRANCH) $(LATEST_TAG)
 
+.PHONY: release-raw-ubuntu-2004
+release-raw-ubuntu-2004: MAKEFLAGS=
+release-raw-ubuntu-2004: export IMAGE_FORMAT=raw
+release-raw-ubuntu-2004: AMI_ID?=ami-0bd99362b6ccd24ea
```

Comment:
is ami id going to overwritten in the build or how is this going to be kept up to date?

Reply:
This is just a public AMI for testing creation in presubmit. The actual AMI for the build will come from a Codebuild project env var in CDK